### ISSUE TITLE
Documentation not showing up in Scripts section in RTD

### ISSUE THREAD

**Issue by uzaymacar** [ivadomed/ivadomed#1035]
In the RTD, documentation for a few scripts currently doesn't show up at https://ivadomed.org/scripts.html. The missing scripts are:
* `ivadomed_visualize_transforms`
* `ivadomed_automate_training`
* `ivadomed_prepare_dataset_vertebral_labeling`

As can be seen in `docs/source/scripts.rst`, the documentation for this part is generated with `autofunction`. 

What is interesting is, when I do

```
cd docs
make html
```

as instructed [here](https://github.com/ivadomed/ivadomed/wiki/documentation) and locally look at the docs, **there is no problem**. Somehow this only happens when deployed.

I tried to find a `latest` build that doesn't have this issue from [here](https://readthedocs.org/projects/ivado-medical-imaging/builds/) but I haven't been successful yet.

**Comment by jcohenadad** [comment]
Interesting. [This previous build](https://ivado-medical-imaging--980.org.readthedocs.build/scripts.html) does show all the scripts. I'm looking into this now.

**Comment by jcohenadad** [comment]
There is a bunch of warnings involving `visualize_transforms` (and other scripts that fail to be displayed) in the latest build, which do not appear in build 980 linked above:

<details><summary>Terminal output</summary>

```console
WARNING: autodoc: failed to import module 'loader' from module 'ivadomed.loader'; the following exception was raised:
cannot import name 'int_classes' from 'torch._six' (/home/docs/checkouts/readthedocs.org/user_builds/ivado-medical-imaging/envs/1034/lib/python3.7/site-packages/torch/_six.py)
WARNING: autodoc: failed to import module 'utils' from module 'ivadomed.loader'; the following exception was raised:
cannot import name 'int_classes' from 'torch._six' (/home/docs/checkouts/readthedocs.org/user_builds/ivado-medical-imaging/envs/1034/lib/python3.7/site-packages/torch/_six.py)
WARNING: autodoc: failed to import module 'utils' from module 'ivadomed.object_detection'; the following exception was raised:
cannot import name 'int_classes' from 'torch._six' (/home/docs/checkouts/readthedocs.org/user_builds/ivado-medical-imaging/envs/1034/lib/python3.7/site-packages/torch/_six.py)
WARNING: autodoc: failed to import module 'evaluation' from module 'ivadomed'; the following exception was raised:
cannot import name 'int_classes' from 'torch._six' (/home/docs/checkouts/readthedocs.org/user_builds/ivado-medical-imaging/envs/1034/lib/python3.7/site-packages/torch/_six.py)
WARNING: autodoc: failed to import module 'main' from module 'ivadomed'; the following exception was raised:
cannot import name 'int_classes' from 'torch._six' (/home/docs/checkouts/readthedocs.org/user_builds/ivado-medical-imaging/envs/1034/lib/python3.7/site-packages/torch/_six.py)
WARNING: autodoc: failed to import module 'testing' from module 'ivadomed'; the following exception was raised:
cannot import name 'int_classes' from 'torch._six' (/home/docs/checkouts/readthedocs.org/user_builds/ivado-medical-imaging/envs/1034/lib/python3.7/site-packages/torch/_six.py)
WARNING: autodoc: failed to import module 'training' from module 'ivadomed'; the following exception was raised:
cannot import name 'int_classes' from 'torch._six' (/home/docs/checkouts/readthedocs.org/user_builds/ivado-medical-imaging/envs/1034/lib/python3.7/site-packages/torch/_six.py)
WARNING: autodoc: failed to import module 'transforms' from module 'ivadomed'; the following exception was raised:
cannot import name 'int_classes' from 'torch._six' (/home/docs/checkouts/readthedocs.org/user_builds/ivado-medical-imaging/envs/1034/lib/python3.7/site-packages/torch/_six.py)
WARNING: autodoc: failed to import module 'visualize' from module 'ivadomed'; the following exception was raised:
cannot import name 'int_classes' from 'torch._six' (/home/docs/checkouts/readthedocs.org/user_builds/ivado-medical-imaging/envs/1034/lib/python3.7/site-packages/torch/_six.py)
WARNING: autodoc: failed to import module 'inference' from module 'ivadomed'; the following exception was raised:
cannot import name 'int_classes' from 'torch._six' (/home/docs/checkouts/readthedocs.org/user_builds/ivado-medical-imaging/envs/1034/lib/python3.7/site-packages/torch/_six.py)
WARNING: autodoc: failed to import function 'visualize_transforms.run_visualization' from module 'ivadomed.scripts'; the following exception was raised:
cannot import name 'int_classes' from 'torch._six' (/home/docs/checkouts/readthedocs.org/user_builds/ivado-medical-imaging/envs/1034/lib/python3.7/site-packages/torch/_six.py)
WARNING: autodoc: failed to import function 'automate_training.automate_training' from module 'ivadomed.scripts'; the following exception was raised:
cannot import name 'int_classes' from 'torch._six' (/home/docs/checkouts/readthedocs.org/user_builds/ivado-medical-imaging/envs/1034/lib/python3.7/site-packages/torch/_six.py)
WARNING: autodoc: failed to import function 'automate_training.HyperparameterOption' from module 'ivadomed.scripts'; the following exception was raised:
cannot import name 'int_classes' from 'torch._six' (/home/docs/checkouts/readthedocs.org/user_builds/ivado-medical-imaging/envs/1034/lib/python3.7/site-packages/torch/_six.py)
WARNING: autodoc: failed to import function 'automate_training.get_param_list' from module 'ivadomed.scripts'; the following exception was raised:
cannot import name 'int_classes' from 'torch._six' (/home/docs/checkouts/readthedocs.org/user_builds/ivado-medical-imaging/envs/1034/lib/python3.7/site-packages/torch/_six.py)
WARNING: autodoc: failed to import function 'automate_training.make_config_list' from module 'ivadomed.scripts'; the following exception was raised:
cannot import name 'int_classes' from 'torch._six' (/home/docs/checkouts/readthedocs.org/user_builds/ivado-medical-imaging/envs/1034/lib/python3.7/site-packages/torch/_six.py)
WARNING: autodoc: failed to import function 'automate_training.update_dict' from module 'ivadomed.scripts'; the following exception was raised:
cannot import name 'int_classes' from 'torch._six' (/home/docs/checkouts/readthedocs.org/user_builds/ivado-medical-imaging/envs/1034/lib/python3.7/site-packages/torch/_six.py)
WARNING: autodoc: failed to import function 'prepare_dataset_vertebral_labeling.extract_mid_slice_and_convert_coordinates_to_heatmaps' from module 'ivadomed.scripts'; the following exception was raised:
cannot import name 'int_classes' from 'torch._six' (/home/docs/checkouts/readthedocs.org/user_builds/ivado-medical-imaging/envs/1034/lib/python3.7/site-packages/torch/_six.py)
/home/docs/checkouts/readthedocs.org/user_builds/ivado-medical-imaging/checkouts/1034/docs/source/tutorials/one_class_segmentation_2d_unet.rst:2: WARNING: Duplicate explicit target name: "here".
```

</details>

**Comment by jcohenadad** [comment]
The oldest build where some scripts are not shown is [build 1033](https://readthedocs.org/projects/ivado-medical-imaging/builds/15615038/).

**Comment by jcohenadad** [comment]
The only suspicious difference between build 1033 (failed) and the previous build (worked) is this added ref:

https://github.com/ivadomed/ivadomed/blob/5df42ddc270e7e4a16f87fc5802657a3d32f9e5a/requirements_common.txt#L23

I am wondering if installing torchio somewhat screwed some other dependencies

**Comment by jcohenadad** [comment]
Since this is not high priority, let's leave it for now. Someone from @dyt811 team could look at it at their convenient schedule.

**Comment by konantian** [comment]
This issue has been resolved in PR #1104. The root cause is after we upgraded `torchio` to a newer version, it does not support importing int_classes from torch._six anymore and caused a bunch of warnings in [build 1033](https://readthedocs.org/projects/ivado-medical-imaging/builds/15615038/).
